src/openssl/ssl/context.cr

```diff
@@ -330,36 +320,27 @@ abstract class OpenSSL::SSL::Context
   end
 
   # Sets the current ciphers and ciphers suites to **modern** compatibility level as per Mozilla
-  # recommendations. See `CIPHERS_MODERN` and `CIPHER_SUITES_MODERN`. See `#security_level=` for some
-  # sensible system configuration.
+  # recommendations. See `#security_level=` for some sensible system configuration.
+  #
+  # Does nothing as per Crystal 1.13.
```

Comment:
In fact, IIUC, before 1.13 it _also_ did nothing, and this is why we're removing it. Shouldn't we mention that?

Reply:
Not exactly. I didn't work with OpenSSL 1.1+ but it did work for OpenSSL 1.0

Reply:
Then let's mention that?